src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -183,6 +183,41 @@ public sealed class SetEnvCommandExtension : RunnerService, IActionCommandExtens
 
         public void ProcessCommand(IExecutionContext context, string line, ActionCommand command, ContainerInfo container)
         {
+
+            var serverUrl = context.GetGitHubContext("server_url");
+            var isGHES = false;
+            if(!String.IsNullOrWhiteSpace(serverUrl) && serverUrl != "https://github.com")
+            {
+                isGHES = true;
+            }
+            
+            var allowUnsecureCommands = false;
+            if (!String.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable(Constants.Variables.Actions.AllowUnsupportedCommands)) 
+                && Environment.GetEnvironmentVariable(Constants.Variables.Actions.AllowUnsupportedCommands).ToUpper() == "TRUE")
+                {
+                    allowUnsecureCommands = true;
+                }
+            // TODO REMOVE DEBUG
```

Comment:
Do you need to remove this? :)

Reply:
It is removed now! Was still testing GHES!